.rubocop.yml

```diff
@@ -2,6 +2,7 @@ AllCops:
   TargetChefVersion: 15.latest
   Exclude:
     - 'cookbooks/**/*'
+    - 'test/integration/**/*'
```

Comment:
Why was this added?

Reply:
There are tabs in the integration tests for haproxy_osuosl, haproxy_phpbb, and mysql2. When I replace the tabs with spaces, the InSpec tests fail.

Reply:
So instead of excluding the whole files I recommend you only disable the check for the sections of code it complains about by doing it this way:
```ruby
# rubocop:disable Layout/IndentationStyle
bad code
# rubocop:enable Layout/IndentationStyle
```
You'll have to do this in multiple places in the same file.